autorandr.py

```diff
@@ -957,8 +957,8 @@ def fork_child_autorandr(pwent, process_environ):
 
         process_environ = {}
         for environ_entry in open(environ_file).read().split("\0"):
-            if "=" in environ_entry:
-                name, value = environ_entry.split("=", 1)
+            name, _, value = environ_entry.partition("=")
+            if name:
```

Comment:
`"foo".partition("=") == ("foo", "", "")`. So this should check for `_ == "="` and `name` rather than only for `name`.

Reply:
Hmm.. I do not think there would be something without `=` normally in `environ`, and if so then I could imagine it meaning an empty var?!
Looking at the cpython source it mentions however that hidden variables on Windows start with `=` - but that is not supported with this code anyway.
Amended.

Reply:
There shouldn't normally be entries that start with a = on Linux either.. alas, technically, the environment is just a list of null terminated strings, so anything can happen :) Thanks, I'll merge this tomorrow.